app/models/version.rb

```diff
@@ -315,7 +315,11 @@ def authors_array
   end
 
   def sha256_hex
-    sha256.unpack("m0").first.unpack("H*").first if sha256
+    Version._sha256_hex(sha256) if sha256
+  end
+
+  def self._sha256_hex(raw)
```

Comment:
It is more about aesthetic and readability than syntax issue. Open an issue about why is this unnecessary and I will send a PR :wink: